README.md

```diff
@@ -5,89 +5,165 @@ OCM Stateful application samples, including Ramen resources.
+## Sample applications
+
+This repository provides samples that can be deployed on Kubernetes and
+OpenShift:
+
+- core
+  - deployment - busybox deployment
+
+
+## Deploying a sample application
+
+In the example we use the busybox deployment for Kubernetes regional DR
+environment using RBD storage:
+
+    subscription/core/deployment-k8s-regional-rbd
+
+This application is deployed in the `deployment-rbd` namespace on the
+hub and managed clusters.
+
+You can use other overlays to deploy on other cluster types or use
+different storage class. You can also create your own overlays based on
+the examples.
+
+1. Deploy an OCM application subscription on hub:
+
+   ```
+   kubectl apply -k subscription/core/deployment-k8s-regional-rbd
+   ```
+
+   This creates the required Subscription, Placement, and
+   ManagedClusterSetBinding resources for the deployment in the
+   `deployment-rbd` namespace and can be viewed using:
+
+   ```
+   kubectl get subscription,placement -n deployment-rbd
+   ```
+
+1. Inspect subscribed resources from the channel created in the same
+   namespace on the ManagedCluster selected by the Placement.
+
+   The busybox deployment Placement `status` can be viewed on the hub
+   using:
+
+   ```
+   kubectl get placement placement -n deployment-rbd
+   ```
+
+   The Busybox deployment subscribed resources, like the pod and the PVC
+   can be viewed on the ManagedCluster using (example ManagedCluster
+   `cluster1`):
+
+   ```
+   kubectl get pod,pvc -n deployment-rbd --context cluster1
+   ```
+
+## Undeploying a sample application
+
+To undeploy an application delete the subscription overlay used to
+deploy the application:
+
+```
+kubectl delete -k subscription/core/deployment-k8s-regional-rbd
+```
+
+## Enable DR for a deployed application
 
 1. Change the Placement to be reconciled by Ramen
 
-    ```
-    kubectl annotate placement busybox-placement -n busybox-sample \
-       cluster.open-cluster-management.io/experimental-scheduling-disable=true
-    ```
+   ```
+   kubectl annotate placement placement -n deployment-rbd \
+      cluster.open-cluster-management.io/experimental-scheduling-disable=true
+   ```
 
 1. Deploy a DRPlacementControl resource for the OCM application on the
    hub, for example:
-    - `kubectl apply -k dr/`
-    - The above creates a DRPlacementControl resource for the busybox
-    application in the `busybox-sample` namespace and can be viewed
-    using:
-        - `kubectl get drpc -n busybox-sample`
-    - At this point the application is managed by *Ramen*.
 
-## Disable DR for the OCM application
+   ```
+   kubectl apply -k dr/core/deployment-k8s-regional-rbd
+   ```
+
+   This creates a DRPlacementControl resource for the busybox deployment
+   in the `deployment-rbd` namespace and can be viewed using:
+
+   ```
+   kubectl get drpc -n deployment-rbd
+   ```
 
-1. Ensure Placement is pointing to the cluster where the workload is
+   At this point the application is managed by *Ramen*.
```

Comment:
The old statement can be better, this should be something like, `At this point the placement of the application is managed by *Ramen*`

Reply:
Fixed in next version.